contrib/modelci/README.md

```diff
@@ -0,0 +1,47 @@
+# Model CI
+
+A simple Continuous Interation for Models, tracking the overall effect and performance.
```

Comment:
What is the plan to run this CI? Are we going to bridge this CI with TeamCity, or set it up as a new configuration on Travis-CI?

Reply:
In the beginning, this is just a bunch of scripts, no relation with TeamCity or other CI platforms.

It might be integrated with TeamCity latter, but currently, just plan to be a while-loop process which keeps testing the last merged code and tracking the performance and precision.

Reply:
I am working on it, seems just a few days work. It will keep clear in the begging, and more factors that need to track can be added later.

But it needs some computation resources, such as a free GPU machine, to test several classical models both in CPU and GPU mode (single card).
@wangkuiyi